programs/test/zeroize.c

```diff
@@ -40,7 +38,6 @@
 #if defined(MBEDTLS_PLATFORM_C)
 #include "mbedtls/platform.h"
 #else
-#include <stdio.h>
```

Comment:
Why this change in the backport but not the main PR? It looks off-topic (though I guess harmless).

Reply:
As mentioned in the commit message for the first commit, this line was never present in the development branch. It was erroneously added to the file, causing the two branches to diverge.
This also changed the line number hardcoded in the GDB script, causing my patch to not apply cleanly to the 2.16 branch.

Reply:
Thanks - I failed to look at the individual commits before, just the overall change.